README.md

```diff
@@ -94,7 +94,7 @@ directory.
   Go implementation of gRPC, gRPC-Web, and Connect
 * [examples-go](https://github.com/connectrpc/examples-go):
   Example RPC service powering https://demo.connectrpc.com and built with connect-go
-* [connect-crosstest](https://github.com/bufbuild/connect-crosstest):
+* [connect-conformance](https://github.com/connectrpc/conformance):
```

Comment:
Should the link be labelled `conformance` instead of `connect-conformance`?

Reply:
maybe, yeah. how about if we make it just `conformance` in all repos in `connectrpc` and `connect-conformance` in others? we sort of do the same with `examples-es` in the Protobuf-ES repo.